### ISSUE TITLE
Install script touch ups

### ISSUE THREAD

**Issue by danielpclark** [danielpclark/abrute#9]
@baptistecolin I've tried this script on a clean Docker image and it doesn't get past `rustc`.

```
docker run -i -t ubuntu /bin/bash
apt-get update
apt-get install curl
curl https://raw.githubusercontent.com/danielpclark/abrute/master/install.sh | bash
```

Output:

```
bash: line 3: rustc: command not found
bash: line 16: [: ==: unary operator expected
```

**Comment by danielpclark** [comment]
Maybe using `which rustc` to see if that produces a value would be better?

**Comment by danielpclark** [comment]
Would it be okay with you if I assign this to you?  I can label it Hacktoberfest for you if you're trying to get the free shirt.

**Comment by baptistecolin** [comment]
oops, I have to fix that indeed.
I will take a look at this shortly.

**Comment by baptistecolin** [comment]
I made a PR to fix this : #10
